MANIFEST.in

```diff
@@ -5,6 +5,10 @@ prune .eggs
 
 # prune eigen LGPL2
```

Comment:
This comment belongs on the `exclude third_party/eigen/Eigen/Eigen` stuff below.

Reply:
moved gloo related stuff below another comment.